app.py

```diff
@@ -62,9 +62,9 @@
 
     if run_mode == 'DEV':
         # For use with Localstack
-        endpoint_url = app_config['AWS_ENDPOINT']
+        endpoint_url = app_config['AWS_ENDPOINT_URL']
```

Comment:
Ok, so we need to change that in our dev envs too after this is merged.

Reply:
(Why the name change btw?)

Reply:
`AWS_ENDPOINT_URL` is the variable used in the [awscli](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-list)